sections/currentState.tex

```diff
@@ -0,0 +1,31 @@
+\section{Current state}
+
+As previously mentioned, \knox{} was initially started in 2020.
+The previous group chose to develop this layer using Java.
+
+Using the framework Apache Jena in combination with the Fuseki package, they set up a simple database system.
+
+The system has two functionalities. 
+Firstly, it can store knowledge graphs in HDT and convert triples to HDT.
+Secondly, it contains a prototype for counting the number of words in an article.
+There is no API for reading this data, only for writing. 
+In addition, the other layers are directly connected to the database.
+
+Overall, not much code was written and its quality is questionable. 
+The Java language convention was not followed, no documentation was made, and only a single test exists. 
+It appears as though no structure was established, making it difficult to navigate the code. 
+Furthermore, the database is set up such that is must be restarted every time new data is written to it - otherwise, the data cannot be fetched.
+
+Based on this, we have discussed what work to do and in what order.
+After having read and understood the current code, we will have to start by cleaning up the inconsistencies and structure.
+We will then have to document what the code does and write tests for it.
```

Comment:
Would be nice if text about server and database was with other text about DB/server. Eventually, everything JAVA could be moved down before what we must do.